src/components/seplanen/sendtilgodkjenning/TvungenGodkjenningToggle.tsx

```diff
@@ -0,0 +1,53 @@
+import { Alert, BodyLong, Radio, RadioGroup } from "@navikt/ds-react";
+import React from "react";
+import styled from "styled-components";
+
+const SpacedAlert = styled(Alert)`
+  margin-bottom: 2rem;
+`;
+
+interface Props {
+  tvungenGodkjenning: boolean | null;
+
+  setTvungenGodkjenning(tvungenGodkjenning: boolean): void;
+}
+
+export const TvungenGodkjenningToggle = ({
+  tvungenGodkjenning,
+  setTvungenGodkjenning,
+}: Props) => {
+  return (
+    <div>
```

Comment:
Usikker på om vi trenger en div-container her for stylingen sin del? Om ikke går det an å bruke [fragments](https://reactjs.org/docs/fragments.html), så sparer vi et dom-element. Bare flisespikkeri :)